src/OrchardCore.Build/Dependencies.props

```diff
@@ -1,7 +1,10 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+  <Import Project="..\..\src\OrchardCore.Build\Dependencies.AspNetCore.props" />
+
   <PropertyGroup>
-    <AspNetCoreVersion>2.0.0</AspNetCoreVersion>
+    <!-- This value is deprecated and will be deleted in beta2-->
+    <AspNetCoreVersion>2.0.1</AspNetCoreVersion>
```

Comment:
Maybe `AspNetCoreVersion` => `MicrosoftAspNetCorePackageVersion`.

Reply:
Not sure, first because it's not part of the dependencies file that is generated by MS, and also because some modules might already be using it, and this work we are not breaking anyone.